tests/framework/ROM/TimeSeries/SyntheticHistory/TrainingData/Clustered_A_0.csv

```diff
@@ -0,0 +1,1001 @@
+pivot,signal0,signal1
+0.000000000000000000e+00,-1.023538297423564752e-01,-1.475257076097549414e+00
+1.000000000000000056e-01,1.882947541575958894e-01,2.134044184248994558e+00
+2.000000000000000111e-01,-1.655719804495711101e-01,-2.284535550657079028e+00
+2.999999999999999889e-01,-3.606511423985305775e-01,-1.602966622035592703e+00
+4.000000000000000222e-01,8.025647150520918149e-01,5.933880922474279451e-01
+5.000000000000000000e-01,1.097985274012541090e+00,2.044384797439171564e-01
+5.999999999999999778e-01,5.954465753781293946e-01,6.307932507459796945e-01
+6.999999999999999556e-01,4.385963641041659145e-01,-5.825439638595943936e-01
+8.000000000000000444e-01,6.038094658580023255e-01,3.474744035754482319e+00
```

Comment:
is there a reasonable way we could test this with a shorter data set? We typically try to keep regression test examples as lean as possible (but not leaner).

Reply:
That's a good point, I forgot this represents all the data that would then be segmented. What is the rough runtime for this test?

Reply:
About 5 seconds on my somewhat dated macbook air.

Reply:
Okay, that's trending towards longer, but not bad given the problem. I think it's fine.